activemq/SMART_AGENT_MONITOR.md

```diff
@@ -19,7 +19,7 @@ Below is an overview of that monitor.
 
 
 SignalFx's integration with ActiveMQ wraps the [Collectd GenericJMX
-monitor](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-genericjmx.html)
+monitor](https://docs.splunk.com/Observability/gdi/genericjmx/genericjmx.html)
```

Comment:
Can we lowercase all this `Observability`?

Reply:
This is how the redirect was set up, I can do that.

Reply:
Please review https://github.com/signalfx/signalfx-agent/pull/2667 :)

Reply:
now fixed in this PR as well.